packages/apm/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const apm = require('elastic-apm-node');
+
+let active = process.env.ELASTIC_APM_ACTIVE;
```

Comment:
Should this default to `false` such that if the package is included without this variable, it won't cause any harm?

Reply:
@smebberson On L8, if it is undefined then it will use `process.env.NODE_ENV === 'production'`

Reply:
@allanchau, yeah, but it means it could be active without the other 'required' environment variables being set?

Reply:
@smebberson so currently apm is only active if `ELASTIC_APM_ACTIVE=true` or the environment is production. If the production environment doesn't contain `ELASTIC_APM_SERVICE_NAME` then elastic apm package should flag it since it won't be able to connect to anything

> The only required parameter is serviceName. However, the agent will use the name from package.json by default if available.

Is this what we want, or do you think it is better to have it silently fail (turn it off).

Reply:
Okay, I think has we have it is okay then.